src/components/molecules/Visualizer/Widget/Navigator/index.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { Meta, Story } from "@storybook/react";
+
+import { Provider } from "../../storybook";
```

Comment:
For paths that are nested more than one level (aka the parent directory) please use the @reearth alias and avoid relative paths

Reply:
Sorry for some reason I think I thought it was a component import, not related to storybook. My bad